packages/react-textarea/src/components/TextArea/useTextAreaStyles.ts

```diff
@@ -0,0 +1,193 @@
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { TextAreaSlots, TextAreaState } from './TextArea.types';
+import { getBottomFocusIndicator } from '@fluentui/react-tabster';
+import { tokens } from '@fluentui/react-theme';
+import { SlotClassNames } from '@fluentui/react-utilities';
+
+export const textAreaClassNames: SlotClassNames<TextAreaSlots> = {
+  root: 'fui-TextArea',
+  textArea: 'fui-TextArea__textArea',
+};
+
+// TODO(sharing) use theme values once available
+const spacingTokens = {
+  horizontal: {
+    sNudge: '6px',
+    mNudge: '10px',
+    xss: '2px',
+    m: '12px',
+  },
+};
+const contentSizeTokens = {
+  body1: {
```

Comment:
I see more and more of these styles that already exist in `react-text` being duplicated... let's bring this up in tech sync @andrefcdias

Reply:
Can react-text be updated to export the constants from packages/react-text/src/typographyStyles/typographyStyles.ts? That would solve most of the problem.

Reply:
The topic is in Wednesday's agenda, I'll bring that up in case it's left out.